app/controllers/idv/in_person/public/usps_locations_controller.rb

```diff
@@ -3,15 +3,35 @@
 module Idv
   module InPerson
     module Public
+      class UspsLocationsError < StandardError
+        def initialize
+          super('Unsupported characters in address field.')
+        end
+      end
+
       class UspsLocationsController < ApplicationController
         skip_forgery_protection
 
+        include IppHelper
+
+        rescue_from ActionController::InvalidAuthenticityToken,
```

Comment:
@jennyverdeyen What would cause an `ActionController::InvalidAuthenticityToken` error to occur?

Reply:
Good question! My understanding is that it occurs when a there is a missing or mismatched CSRF token: https://dev.to/ben/actioncontroller-invalidauthenticitytoken-what-s-going-on-here-2828 

I'm not sure more specifically how it could be triggered in this controller, but I deduced it would be worth catching since it was also being [caught in the other version of this controller ](https://github.com/18F/identity-idp/blob/main/app/controllers/idv/in_person/usps_locations_controller.rb#L24)

Reply:
This controller is public and includes [skip_forgery_protection](https://github.com/18F/identity-idp/pull/11470/files#diff-6828631ffb887869828da62a77bb09e7b885f9ca74e7426d5b44449116c8df2cL7), which means the CSRF token is not being checked. Because of that, it should be safe to remove `ActionController::InvalidAuthenticityToken`.

Reply:
@mitchellhenke Thanks for the explanation, that makes sense! I'll remove it.

Reply:
This was educational, thanks. 🙏🏻